etc/ReplayOfflineConfiguration.py

```diff
@@ -407,9 +413,10 @@
                  data_tiers=["FEVT"],
                  write_dqm=True,
                  alca_producers=["SiStripPCLHistos", "SiStripCalZeroBias", "SiStripCalMinBias", "SiStripCalMinBiasAAG",
-                                 "TkAlMinBias", "LumiPixelsMinBias", "SiPixelCalZeroBias",
+                                 "TkAlMinBias", "SiPixelCalZeroBias","SiPixelCalSingleMuon", "SiPixelCalSingleMuonTight",
```

Comment:
why not `SiPixelCalSingleMuonLoose` as well, then?

Reply:
Hi Marco, in standard pp `SiPixelCalSingleMuonLoose` is applied only on the Muon PD at Prompt level and not at Express level, so I forgot to add it to `HIExpress`. Thanks for checking, I will push a new commit now to include it now.